data/human/variants.txt

```diff
@@ -1368,8 +1368,8 @@ ship "Headhunter" "Headhunter (Strike)"
 	outfits
 		"Javelin Pod" 2
 		"Javelin" 400
-		"Meteor Missile Launcher" 2
-		"Meteor Missile" 60
+		"Meteor Missile Pod" 2
```

Comment:
It is, however I swapped it out for the pods in an effort to have more of a progression for new pilots. Due to the amount of missiles a standard launcher can hold (vastly more than any light ship has a hope of dealing with), interceptors starts (and for that matter, combat in interceptors and other small ships, typically against pirates) are practically impossible. 

So while yes, some light warships (particularly military ones as opposed to civilian ones) should carry full launchers, we still need to have at least *some* of them with pods instead. It comes down to role, though.

Pods:
- Ships aimed at taking out single (usually light) ships
- Ships that are generally "strike and run"
- Ships that operate close to somewhere they can resupply (carrier or station)

Launchers:
- Ships aimed at taking out fleets and/or heavy warships
- Ships that generally "hold the line" and are expected to control a system for long periods of time and/or protracted battles
- Ships that operate at long ranges (aka multiple hyperjumps) from support and are expected to go through many battles before their next resupply.

Keep in mind that there's also a discrepancy in NPC vs Player outfitting.  The player is virtually *always* going to be falling into that last category "operating at long range from support and expecting to go through multiple battles before their next resupply," whereas NPCs don't. I know thematically, many NPCs *would* operate that way, but a) it's expensive and they have a budget, and b) it makes the game a lot harder for the player. (See: All the comments we get about new players getting annihilated by "small" pirate spawns.)

Reply:
Cool, sounds good to me. I'm not too familiar yet with the roles of different ships beyond the official categories in the ships data file, so I will definitely take your word for it.

As a side note, your last paragraph makes me think it might be neat to make it so some NPCs spawn with a quantity of secondary weapon ammo that is randomized in some way -- perhaps dependent on how professional vs. ragtag the fleet in question is supposed to be, and how close to combat zones or far from "home" they are. Seems like it would improve the game thematically like you mentioned, without necessarily making the game harder. Obviously out of scope for this PR, though.